tmp/lanelet2_extension/include/lanelet2_extension/visualization/visualization.hpp

```diff
@@ -241,6 +241,16 @@ visualization_msgs::msg::MarkerArray speedBumpsAsMarkerArray(
   const std::vector<lanelet::SpeedBumpConstPtr> & sb_reg_elems, const std_msgs::msg::ColorRGBA & c,
   const rclcpp::Duration & duration = rclcpp::Duration(0, 0));
 
+/**
+ * [busStopsAsMarkerArray creates marker array to visualize bus stops]
+ * @param  da_reg_elems [bus stop regulatory elements]
```

Comment:
It doesn't affect the execution result in any way, though,
`da_reg_elements` stands for `detection_area_regulatory_elements`.

Reply:
I fixed just in case.
https://github.com/tier4/autoware_common/pull/2/commits/d25b1b5719a0be0626baf6e4678ec3b956862ba7